modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -297,7 +297,10 @@ public void register(Registry registry) {
 			"3.3.0", "3.4.0",
 			new UpgradeCTModel("DDMStructureLink", "DDMTemplateLink"));
 
-		registry.register("3.4.0", "3.5.0", new UpgradeDDMFormInstanceReport());
+		registry.register(
+			"3.4.0", "3.5.0", new UpgradeDDMFormInstanceReport(),
+			new com.liferay.dynamic.data.mapping.internal.upgrade.v3_5_0.
+				UpgradeDDMDataProviderInstance());
```

Comment:
@carolmariaabb, is there any special reason for you use an existent schema upgrade instead of use an exclusive one, like `3.7.0` (the next one available)?